packages/hw-common/src/validation.ts

```diff
@@ -29,6 +29,7 @@ const selectAccountOptions = Joi.object({
   chains: chains,
   scanAccounts: Joi.function().arity(1).required(),
   supportsCustomPath: Joi.bool(),
+  consecutiveEmptyAccountThreshold: Joi.bool(),
```

Comment:
```suggestion
  consecutiveEmptyAccountThreshold: Joi.number(),
```
Should this be set to `number`?